### DISCUSSION TITLE
Twig Block Deferred Stopped Working After Updating Pimcore

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#17082] [Q&A / Support] [answered]
Previously, I had Pimcore 10.5, and to install Datahub and Pimcore Data Importer, I had to update to 10.6. Subsequently, all lines where there is a deferred in Twig throw an error of this type: "Warning: foreach() argument must be of type array|object, null given". The problem is that this syntax exists in bundles like the ecommerce framework, where I encounter the error, and I don't want to modify the template directly in the vendor to solve the error. What can I do?  
![image](https://github.com/pimcore/pimcore/assets/70108013/fa15767f-3a22-40d6-ab62-e9af525f46f7)
![image](https://github.com/pimcore/pimcore/assets/70108013/c196eeb1-4250-4ac1-b314-d4fcef7f0384)

**Comment by AlephLau** [comment]
@fashxp Can you help me with this pls?  🙏

**Reply by dpfaffenbauer** [reply, marked as answer]
add this conflict to your composer.json and run composer update: https://github.com/pimcore/demo/blob/11.x/composer.json#L36

**Reply by AlephLau** [reply]
Thanks a lot! It works

**Reply by mario-neuhold** [reply]
As that link is getting redirected to the 2025.x branch now, I think this one should point to the mentioned conflict: https://github.com/pimcore/demo/blob/4df3ee70febfacda936dc0b70cc0a5e991faca82/composer.json#L36
And just to be "future-proof", here's the relevant code:

```
  "conflict": {
    "twig/twig": "^3.9.0"
  },
```

At least it helped me solve the same as OPs problem.

**Reply by jdreesen** [reply]
Please note that all the versions you'll be able to install with this have security issues.
